tools/topology/topology1/sof-cavs-nocodec.m4

```diff
@@ -107,10 +107,13 @@ dnl     time_domain, sched_comp)
 
 # Low Latency playback pipeline 1 on PCM 0 using max 2 channels of PIPE_BITS.
 # Set 1000us deadline on core 2 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+# For BXT, this will be a regular volume playback pipeline. For all other cavs platforms,
+# this will be replaced by mixer based playback pipeline
```

Comment:
why special-case? it'd be fine to use a mixer on all 3 dai pipelines, and only connect a second host pipeline for one of the mixers.
That way all 3 playback path have the same latency/processing/settings.

Reply:
that would also make the pipeline management less complicated with fewer options.

Reply:
@plbossart fixed now. here's  how the tplg's look now 
APL:
![tplg](https://user-images.githubusercontent.com/7766921/126699815-22bd0600-5ee0-4986-9ac7-48521f5eb83b.png)
Rest:
![tplg](https://user-images.githubusercontent.com/7766921/126699855-5f028d37-5900-4e67-b8f7-2f70a337d2e6.png)

Reply:
just what the doctor ordered. Thanks Ranjani.